.github/workflows/functional-test.yaml

```diff
@@ -0,0 +1,54 @@
+name: Functional test
+
+on:
+  pull_request:
+    paths:
+      - docker/**
+      - charts/**
```

Comment:
Should we add kustomize also?

Reply:
kustomize is deprecated. Right ? It only validate sandbox end 2 end. In sandbox we don't use kustomize now